### ISSUE TITLE
[survival beta - suggestions] Impressions

### ISSUE THREAD

**Issue by Uncle-Ulty** [tsunamayo/Starship-EVO#5496]
I completed all the steps you intended for this beta, and these are my suggestions.

- Gold, Silver and Cobalt ore are hard to look for, since they are rare materials, and this is ok. But, it would be less tedious if we could build a hovercraft before we looked for those minerals. The search for rare veins would be much more interesting with a hovercraft. This can be solved using only iron/stone/wood to build all the requirements for the first hovercraft.

- The refueling process isn't so intuitive in my opinion. I had no idea it would require a canister to refill the ship/hovercraft tank. I kinda overengineered it... I placed a "expel" fuel collector on the electrolysis station,  and a "harvest" fuel collector on the ship to make the fuel transfer. I believe we need the "docking" feature ASAP for this.

- The scrapping process should be faster. Something 2x ~ 4x faster would be ok

- Inventory should hold more items.

-lots of materials such as coal and crystal require description in the menu

-stone and wood should be stackable, or else it will spam the inventory

-imo there are too many wrecks over the planet. Unless it's a result of a combat, it's kinda strange.

-Survival mode is relaxing. The soundtrack and the crafting are a perfect match! I can help you with sound in the future, I'm a musician and I'm learning ableton 11 live. I have bass, midi controller, a guitar, microphone, a Tascam audio interface, and I can do some nice stuff :D.

cheers! This game is getting prettier with each passing day!

**Comment by surajnairpv** [comment]
yeah I agree with the hovercraft suggestion, been stuck with searching for silicon for the hovercraft system, if not hovercraft you can make like a hover bike or hoverboard that requires basic resource as mentioned by @Uncle-Ulty

**Comment by tsunamayo** [comment]
@surajnairpv You can search for a while there is no silicon ore. You need to add 8 rocks to craft silicon. Sorry there is no help at that point but I will make sure it is clear for the player in the future!

**Comment by surajnairpv** [comment]
Ohhhh Maaaan....... I was searching for silicon ore for so long....was thinking how the others were progressing while I am stuck with silicon, been throwing out the rocks in my inventory thinking they had no purpose but never knew you could use to craft silicon from them. Thanks @tsunamayo.

**Comment by BigBadKangaroo** [comment]
Hi there,

I'd like to share my experience as well. I didn't want to creat a new post, so I'll take Ulty's :D 


- I had a lot of trouble finding silver ore. This took me around 30 minutes to find. Something like an Ore Scanner would greatly improve that (I know you're working on that)

- While searching for ore, I noticed that the fastest way to move around and finding ore is to boost straight up in the sky with the jetpack, then look around for ore, and then boost towards each ore I could find. This wasn't the greatest experience however and a possible solution would be to have the anti-g mode back. 

- Scrapping a wreck is way to tedious. Some blocks of the wreck will take 2 seconds to be scrapped, some will take up to 20 seconds. I even had the feeling that not every scrapped block will drop some scrap. Frustrating when one block takes 10+ seconds to be destroyed ...
Something that bugged me a bit was that every block will drop only scrapmetal (if at all). "Special blocks" like lightcruise engine, reactor, glass, pipes, medbays, screens etc. will all drop scrap ... it would be much more interesting if the "special blocks" would drop something special. Just to make scrapping a wreck more rewarding and even more useful except for gathering scrap hull.
I.e. add construction,- electrical,- mechanical components, computer etc. to the drop chance for "special blocks".

- After I got every type of ore to build the assembler (and the other tech after that) everything went pretty okay-ish. There are a ton of things that need a tutorial on how-to-use stuff. How do I connect the furnace / assembler etc. to my storage units? What is that for a block in my inventory where my cursor is pointing at? How could I use that specific block? Just to give an example: I didn't know how to use the Electrolyzer. I tried for like 10 minutes to get it to work because I never used it before in it's supposed use case. There's no mentioning that it needs a fuel scoop to work. I had to check the Stevo Wiki on how to connect everything up to get it to work.

- In general I didn't like the inventory layout when crafting. To make the screen more user friendly while crafting, I'd change it to give each menu it's own space - something like that:
![Stevo Survival Crafting](https://github.com/tsunamayo/Starship-EVO/assets/81495956/98304507-d918-4681-9545-ccc4fa1ae23c)
Maybe the player could be able to drag&drop each window? That's how MMO's deal with the window cluttering on screen. This way every player could organise the screen to their liking.

- In the current state of the game I didn't even bother with a hovercraft. I checked on what components I need to build one and figured out that I could start directly with a spaceship ... it's just not worth it to build a hovercraft right now when you can have a spaceship that will transport you further away, much faster and is easier to build. I've seen you're into adding batteries and I think this will help, but maybe the player should not be able to get the fuel production right away. It should take a bit longer to even get fuel production going. (Perhaps that's when a Tech-Tree comes in handy) Once you have the fuel production there's no reason for me to get back to a hover craft. Even a small production gets you enough fuel for your first trips to space. When you're starting to build the first "big" spaceship that will get you to other planets, you have enough resources to upgrade your fuel production so it will not become an issue.

- Building a ship in survival mode is a pain ... NoClip mode feels like cheating but there's no way around (at least for me!) when building a ship. Maybe we could have a block like the Shipyard block where we can set a dedicated area just for building a ship. Inside of the area the player would noclip through the newly build blueprint to make building more easy. The ship blueprint would also be static and could be rotated inside a small menu when clicking "F" on the block. The player could even be invincible / have unlimited jetpack fuel and oxygen when inside this zone?
This new block could also be tied in to the factions gameplay. I.e. when you want to build a large ship you could fly up to a dry dock spacestation and use the shipyard to build (where it is easier to build than on your planet base). This could then be tied up to the progression of each faction. Let's say the dry dock is blocked until you reach a certain standing with the faction that owns the dry dock.


In general I really like the experience I had with the survival mode so far. Yes, there are a few things that need to be changed, added etc. but the overall feel I have when playing is great. It's a relaxing thing and even quite enjoyable. Now it just needs to be more polished and self-explanatory so even new players understand what they're supposed to do and how stuff works. The framework of it is great so far!
